server/app/app.go

```diff
@@ -254,17 +256,13 @@ func (app *App) Start(opts ...appOption) error {
 		Subrouter()
 
 	registerTransactionResource(transactionsRepository, apiRouter, provisioner, tracer)
-
-	registerConfigResource(configRepo, apiRouter, db, provisioner, tracer)
-
-	registerPollingProfilesResource(pollingProfileRepo, apiRouter, db, provisioner, tracer)
-	registerEnvironmentResource(environmentRepo, apiRouter, db, provisioner, tracer)
-
-	demoRepo := demoresource.NewRepository(db)
-	registerDemosResource(demoRepo, apiRouter, db, provisioner, tracer)
-
-	registerDataStoreResource(dataStoreRepo, apiRouter, db, provisioner, tracer)
-	registerlinterResource(linterRepo, apiRouter, db, provisioner, tracer)
+	registerConfigResource(configRepo, apiRouter, provisioner, tracer)
+	registerPollingProfilesResource(pollingProfileRepo, apiRouter, provisioner, tracer)
+	registerEnvironmentResource(environmentRepo, apiRouter, provisioner, tracer)
+	registerDemosResource(demoRepo, apiRouter, provisioner, tracer)
+	registerDataStoreResource(dataStoreRepo, apiRouter, provisioner, tracer)
+	registerlinterResource(linterRepo, apiRouter, provisioner, tracer)
```

Comment:
the `L` should be uppercase here
```suggestion
	registerLinterResource(linterRepo, apiRouter, provisioner, tracer)
```